pkg/proxy/iptables/proxier_test.go

```diff
@@ -1345,6 +1347,270 @@ func assertIPTablesRulesNotEqual(t *testing.T, expected, result string) {
 	}
 }
 
+// cidrContains checks if cidrStr contains ipStr. cidrStr can actually be a CIDR or an IP
+func cidrContains(cidrStr, ipStr string) bool {
+	ip := netutils.ParseIPSloppy(ipStr)
+	if ip == nil {
+		panic("bad IP in test case: " + ipStr)
+	}
+
+	if strings.Contains(cidrStr, "/") {
+		_, cidr, err := netutils.ParseCIDRSloppy(cidrStr)
+		if err != nil {
+			panic("bad CIDR in kube-proxy output: " + err.Error())
```

Comment:
add the string to the panic message 

bad CIDR in kube-proxy output (%q) = %v

Reply:
the error from ParseCIDR always includes the input string